### PR TITLE
[4.0] Fix icon to display adjacent to field

### PR DESCRIPTION
**Pull request by Quy**
Pull Request for Issue #29742.

### Testing Instructions
Edit the login module
Set `Display Labels` = text
Go to front end

### Actual result BEFORE applying this Pull Request
29742-before

### Expected result AFTER applying this Pull Request
29742-after

### CODE REVIEW

**Comment by brianteeman** [comment]
using title to display help text is not only ugly but its not accessible

**Comment by particthistle** [comment]
I have tested this item :white_check_mark: successfully on f3b42c18682af840670e57f51c835804aa246756<br /><br />Layout issue that the PR is designed to resolve is corrected by the PR.

That said, looking at the code, I agree with @brianteeman, and the entire Login Form module is missing accessibility attributes for all the fields, so that's something for a different PR.

Additionally, I noticed for the first time today the "Web Authentication" button after turning on Two Factor Authentication to test this PR. Like 2FA, should this also not be turned on by default when Joomla 4 is installed? If it shouldn't, another PR.

<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30897">issues.joomla.org/tracker/joomla-cms/30897</a>.</sub>

**Comment by chmst** [comment]
I agree with @brianteeman that the title attribute is contra a11y. You can add a tooltip instead.

**Comment by adj9** [comment]
I have tested this item :white_check_mark: successfully on f3b42c18682af840670e57f51c835804aa246756<br /><br />Done :)
OK! <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30897">issues.joomla.org/tracker/joomla-cms/30897</a>.</sub>

**Comment by Scrabble96** [comment]
> 
> 
>  You can add a tooltip instead.

Tooltips don't show on iPad Safari and probably not on Android devices not using a mouse, either.

**Comment by ChrisHoefliger** [comment]
Tested successfully

**Comment by richard67** [comment]
> I agree with @brianteeman that the title attribute is contra a11y. You can add a tooltip instead.

@brianteeman @chmst Does it mean we should not set this PR to RTC? Or does it mean it should be changed with another, future PR?

**Comment by gostn** [comment]
need label:"Information Required"

**Review by N6REJ — CHANGES_REQUESTED** [review]
`fas fa-question fa-fw` should be `icon-question icon-fw`

**Comment by ghost** [comment]
who care on drafts having two successfully test?

**Comment by richard67** [comment]
@gostn It's the author who decides if a PR is draft or not.

**Comment by ghost** [comment]
@Quy what now, test are wastet time?

**Comment by HLeithner** [comment]
> Quy what now, test are wastet time?

@gostn claim down, PRs may involve over time and if Quy founds something that's not right or have to be changed because of other PRs are merged in the meantime it's good that the PR opener sets the PR to draft.

**Comment by ghost** [comment]
i dont make pressure, but at issutracker i list pr having 2 successfully test but not rtc – draft isnt shown in this list:

Screen Shot 2020-12-01 at 11 43 20

**Comment by Quy** [comment]
Sorry I don't have much time these days to contribute.

@gostn Accessibility to be fixed per Brian https://github.com/joomla/joomla-cms/pull/30897#issuecomment-702997741
See https://github.com/joomla-extensions/patchtester/issues/286 regarding Draft status.

File: modules/mod_login/tmpl/default.php [hunk 1]
```diff
@@ -80,16 +80,20 @@
...
 					<label for="modlgn-secretkey-<?php echo $module->id; ?>"><?php echo Text::_('JGLOBAL_SECRETKEY'); ?></label>
-					<input id="modlgn-secretkey-<?php echo $module->id; ?>" autocomplete="one-time-code" type="text" name="secretkey" class="form-control" placeholder="<?php echo Text::_('JGLOBAL_SECRETKEY'); ?>">
-					<span class="btn width-auto" title="<?php echo Text::_('JGLOBAL_SECRETKEY_HELP'); ?>">
-						<span class="fas fa-question" aria-hidden="true"></span>
-					</span>
+					<div class="input-group">
+						<input id="modlgn-secretkey-<?php echo $module->id; ?>" autocomplete="one-time-code" type="text" name="secretkey" class="form-control" placeholder="<?php echo Text::_('JGLOBAL_SECRETKEY'); ?>">
+						<span class="input-group-append" title="<?php echo Text::_('JGLOBAL_SECRETKEY_HELP'); ?>">
+							<span class="input-group-text">
+								<span class="fas fa-question fa-fw" aria-hidden="true"></span>
```
Comment (N6REJ) [inline modules/mod_login/tmpl/default.php:93]: <span class="fas fa-question fa-fw" aria-hidden="true"></span> should be
<span class="icon-question icon-fw" aria-hidden="true"></span>

**Review by N6REJ — CHANGES_REQUESTED** [review]
`<span class="fas fa-question fa-fw" aria-hidden="true"></span>` should be
`<span class="icon-question icon-fw" aria-hidden="true"></span>`

File: modules/mod_login/tmpl/default.php [hunk 2]
```diff
@@ -80,16 +80,20 @@
 						<input id="modlgn-secretkey-<?php echo $module->id; ?>" autocomplete="one-time-code" type="text" name="secretkey" class="form-control" placeholder="<?php echo Text::_('JGLOBAL_SECRETKEY'); ?>">
 						<span class="input-group-append" title="<?php echo Text::_('JGLOBAL_SECRETKEY_HELP'); ?>">
 							<span class="input-group-text">
-								<span class="fas fa-question" aria-hidden="true"></span>
+								<span class="fas fa-question fa-fw" aria-hidden="true"></span>
```
Comment (Quy) [inline modules/mod_login/tmpl/default.php]:
```suggestion
								<span class="icon-question icon-fw" aria-hidden="true"></span>
```

File: modules/mod_login/tmpl/default.php [hunk 3]
```diff
@@ -80,16 +80,20 @@
...
 					<label for="modlgn-secretkey-<?php echo $module->id; ?>"><?php echo Text::_('JGLOBAL_SECRETKEY'); ?></label>
-					<input id="modlgn-secretkey-<?php echo $module->id; ?>" autocomplete="one-time-code" type="text" name="secretkey" class="form-control" placeholder="<?php echo Text::_('JGLOBAL_SECRETKEY'); ?>">
-					<span class="btn width-auto" title="<?php echo Text::_('JGLOBAL_SECRETKEY_HELP'); ?>">
-						<span class="fas fa-question" aria-hidden="true"></span>
-					</span>
+					<div class="input-group">
+						<input id="modlgn-secretkey-<?php echo $module->id; ?>" autocomplete="one-time-code" type="text" name="secretkey" class="form-control" placeholder="<?php echo Text::_('JGLOBAL_SECRETKEY'); ?>">
+						<span class="input-group-append" title="<?php echo Text::_('JGLOBAL_SECRETKEY_HELP'); ?>">
+							<span class="input-group-text">
+								<span class="fas fa-question fa-fw" aria-hidden="true"></span>
```
Comment (Quy) [inline modules/mod_login/tmpl/default.php]:
```suggestion
								<span class="icon-question icon-fw" aria-hidden="true"></span>
```

**Comment by infograf768** [comment]
conflicts

**Comment by infograf768** [comment]
If I choose icons instead of display text (after correcting conflicts), I get
Screen Shot 2020-12-04 at 09 41 29

Is that intended?

**Comment by ashvini77** [comment]
I have tested this item :white_check_mark: successfully on 13dbfa615e4c8acb3f93de3db7a623c70815070d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30897">issues.joomla.org/tracker/joomla-cms/30897</a>.</sub>

**Comment by Scrabble96** [comment]
When the Cassiopeia updates were proposed in PR #31520 there was a comment (I can't find it now) suggesting that the checkbox size (next to 'remember me') was enlarged. I suggested the following code and now also include the RTL version. Perhaps this could be included with this PR if suitable?

`/* ltr */`
`.com-users-login fieldset .control-label {  margin-left: 2rem; }`
`.com-users-login fieldset input[type="checkbox"].inputbox { position: relative; transform: scale(1.5);  top: -2rem; }`

`/* rtl */`
`.com-users-login fieldset .control-label {  margin-right: 2rem; }`

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 13dbfa615e4c8acb3f93de3db7a623c70815070d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30897">issues.joomla.org/tracker/joomla-cms/30897</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30897">issues.joomla.org/tracker/joomla-cms/30897</a>.</sub>

**Comment by Quy** [comment]
@Scrabble96 Thank you for the code. It should be done in a separate PR since it is not related.

This PR fixes the alignment only. Let's fix the ay11 issue in a separate PR.

**Comment by drmenzelit** [comment]
Thanks

**Pull request merged**